### ISSUE TITLE
Fix `stunner-editors` broken tests and re-enable them on kie-tools

### ISSUE THREAD

**Issue by thiagoelg** [apache/incubator-kie-issues#1431]
On 2024-08-07 some tests stopped working on `stunner-editors` in kie-tools due to the following (or similar) error:

> java.lang.NullPointerException: Cannot invoke "org.kie.workbench.common.stunner.bpmn.client.forms.fields.i18n.StunnerFormsClientFieldsConstants.Data_Inputs()" because "org.kie.workbench.common.stunner.bpmn.client.forms.fields.i18n.StunnerFormsClientFieldsConstants.CONSTANTS" is null

This task aims to fix these tests and re-enable them.

It's worth noting that it was not possible to reproduce these issues locally, as building and running tests worked normally.

List of broken and now disabled tests (from `kie-wb-common-stunner/kie-web-common-stunner-sets/kie-web-common-stunner-bpmn/kie-wb-common-stunner-bpmn-client`):
- testSetPossibleOutputAssignmentsDataTypes
- testSetPossibleOutputAssignmentsVariables
- testSetInputAssignmentsDisallowedNames
- testSetPossibleInputAssignmentsDataTypes
- testSetPossibleInputAssignmentsVariables
- buildDefaultImportsCountString
- buildWSDLImportsCountString
- buildImportsCountString
- setImportsCount
- setValue
- testInitWidget
- testAttributeChangeHandlerWhenNotDuplicate
- testAttributeChangeHandlerWhenDuplicate
- testSetVariableCountsString1
- testSetVariableCountsString2
- testSetVariableCountsString3
- testSetVariableCountsString4
- testSetVariableCountsString5
- testSetVariableCountsString6
- testSetVariableCountsString10
- testSetVariableCountsString11

**Comment by jomarko** [comment]
Thank you for reporting the issue. One thing I decided to check is the history of the changes in the `stunner-editors` package https://github.com/apache/incubator-kie-tools/tree/main/packages/stunner-editors. We can see there are basically no changes for months. The last changed file that somehow could change the build results is the pom.xml, however also that change is ~1month old. That brings me to an idea if the github action environment wasn't changed somehow, I mean maybe some java or maven version is now different on the ubuntu machine we use for build via github action?

**Comment by yesamer** [comment]
I agree with @jomarko, 3 days ago a new Ubuntu image was released https://github.com/actions/runner-images/releases/tag/ubuntu24%2F20240804.1 for GitHub action, most likely there's some incompatibility with a tool / software version

**Comment by thiagoelg** [comment]
I believe that `ubuntu-latest` still points to Ubuntu 22.04 (which also received an update: https://github.com/actions/runner-images/releases/tag/ubuntu22%2F20240804.1)

**Comment by yesamer** [comment]
I guess it's time to close this, as we removed the stunner editors :)
